### ISSUE TITLE
1 of 1 test failed

### ISSUE THREAD

**Issue by damon-kwok** [libuv/libuv#2077]
* **Version**: git-master 2018-11-15
* **Platform**: cygwin
````
# uname -a
CYGWIN_NT-10.0 DESKTOP-EKK7E17 2.11.1(0.329/5/3) 2018-09-05 10:24 x86_64 Cygwin
````

<details>

```
+ libtoolize --copy

+ aclocal -I m4

+ autoconf

+ automake --add-missing --copy

checking for a BSD-compatible install... /usr/bin/install -c

checking whether build environment is sane... yes

checking for a thread-safe mkdir -p... /usr/bin/mkdir -p

checking for gawk... gawk

checking whether make sets $(MAKE)... yes

checking whether make supports nested variables... yes

checking build system type... x86_64-unknown-cygwin

checking host system type... x86_64-unknown-cygwin

checking for gcc... gcc

checking whether the C compiler works... yes

checking for C compiler default output file name... a.exe

checking for suffix of executables... .exe

checking whether we are cross compiling... no

checking for suffix of object files... o

checking whether we are using the GNU C compiler... yes

checking whether gcc accepts -g... yes

checking for gcc option to accept ISO C89... none needed

checking whether gcc understands -c and -o together... yes

checking for style of include used by make... GNU

checking dependency style of gcc... gcc3

checking if gcc supports -pedantic flag... yes

checking for gcc way to treat warnings as errors... -Werror

checking if gcc supports -fvisibility=hidden... yes

checking if gcc supports -g flag... yes

checking if gcc supports -std=gnu89 flag... yes

checking if gcc supports -Wall flag... yes

checking if gcc supports -Wextra flag... yes

checking if gcc supports -Wno-unused-parameter flag... yes

checking if gcc supports -Wstrict-prototypes flag... yes

checking for ar... ar

checking the archiver (ar) interface... ar

checking how to print strings... printf

checking for a sed that does not truncate output... /usr/bin/sed

checking for grep that handles long lines and -e... /usr/bin/grep

checking for egrep... /usr/bin/grep -E

checking for fgrep... /usr/bin/grep -F

checking for ld used by gcc... /usr/x86_64-pc-cygwin/bin/ld.exe

checking if the linker (/usr/x86_64-pc-cygwin/bin/ld.exe) is GNU ld... yes

checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B

checking the name lister (/usr/bin/nm -B) interface... BSD nm

checking whether ln -s works... yes

checking the maximum length of command line arguments... 8192

checking how to convert x86_64-unknown-cygwin file names to x86_64-unknown-cygwin format... 
func_convert_file_noop

checking how to convert x86_64-unknown-cygwin file names to toolchain format... func_convert_file_noop

checking for /usr/x86_64-pc-cygwin/bin/ld.exe option to reload object files... -r

checking for objdump... objdump

checking how to recognize dependent libraries... file_magic ^x86 archive import|^x86 DLL
checking for dlltool... dlltool

checking how to associate runtime and link libraries... func_cygming_dll_for_implib

checking for archiver @FILE support... @

checking for strip... strip

checking for ranlib... ranlib

checking command to parse /usr/bin/nm -B output from gcc object... ok

checking for sysroot... no

checking for a working dd... /usr/bin/dd

checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1

checking for mt... no

checking if : is a manifest tool... no

checking how to run the C preprocessor... gcc -E

checking for ANSI C header files... yes

checking for sys/types.h... yes

checking for sys/stat.h... yes

checking for stdlib.h... yes

checking for string.h... yes

checking for memory.h... yes

checking for strings.h... yes

checking for inttypes.h... yes

checking for stdint.h... yes

checking for unistd.h... yes

checking for dlfcn.h... yes

checking for objdir... .libs

checking if gcc supports -fno-rtti -fno-exceptions... no

checking for gcc option to produce PIC... -DDLL_EXPORT -DPIC

checking if gcc PIC flag -DDLL_EXPORT -DPIC works... yes

checking if gcc static flag -static works... yes

checking if gcc supports -c -o file.o... yes

checking if gcc supports -c -o file.o... (cached) yes

checking whether the gcc linker (/usr/x86_64-pc-cygwin/bin/ld.exe) supports shared libraries... yes

checking whether -lc should be explicitly linked in... yes

checking dynamic linker characteristics... Win32 ld.exe

checking how to hardcode library paths into programs... immediate

checking whether stripping libraries is possible... yes

checking if libtool supports shared libraries... yes

checking whether to build shared libraries... yes

checking whether to build static libraries... yes

checking whether make supports nested variables... (cached) yes

checking for dlopen in -ldl... yes

checking for kstat_lookup in -lkstat... no

checking for gethostbyname in -lnsl... no

checking for perfstat_cpu in -lperfstat... no

checking for pthread_mutex_init in -lpthread... yes

checking for clock_gettime in -lrt... yes

checking for sendfile in -lsendfile... no

checking for socket in -lsocket... no

checking for special C compiler options needed for large files... no

checking for _FILE_OFFSET_BITS value needed for large files... no

checking sys/ahafs_evProds.h usability... no

checking sys/ahafs_evProds.h presence... no

checking for sys/ahafs_evProds.h... no

checking that generated files are newer than configure... done

configure: creating ./config.status

config.status: creating Makefile

config.status: creating libuv.pc

config.status: executing depfiles commands

config.status: executing libtool commands

make: Nothing to be done for 'all'.

make  test/run-tests.exe

make[1]: Entering directory '/home/damon/projects/c/libuv'

make[1]: 'test/run-tests.exe' is up to date.

make[1]: Leaving directory '/home/damon/projects/c/libuv'

make  check-TESTS

make[1]: Entering directory '/home/damon/projects/c/libuv'

1..342

not ok 1 - platform_output

# **exit code 134**

# Output from process `platform_output`:

# uv_get_process_title:

# uv_cwd: /home/damon/projects/c/libuv

# Assertion failed in test/test-platform-output.c on line 53: err == UV_ENOSYS

ok 2 - active

ok 3 - async

ok 4 - async_null_cb

ok 5 - async_ref

ok 6 - barrier_1

ok 7 - barrier_2

ok 8 - barrier_3

not ok 9 - barrier_serial_thread

# **exit code 6**

# Output from process `barrier_serial_thread`: (no output)

ok 10 - barrier_serial_thread_single

not ok 11 - callback_stack

# timeout

# Output from process `callback_stack`: (no output)

Output from process `callback_stack`:

# Connecting...

# Connected. Write some data to echo server...

# Data written. 500ms timeout...

# Timeout complete. Now read data...

ok 12 - check_ref

ok 13 - close_fd

ok 14 - close_order

ok 15 - closed_fd_events

ok 16 - condvar_1

ok 17 - condvar_2

ok 18 - condvar_3

ok 19 - condvar_4

ok 20 - condvar_5

ok 21 - connect_unspecified

ok 22 - connection_fail

ok 23 - connection_fail_doesnt_auto_close

ok 24 - cwd_and_chdir

ok 25 - default_loop_close

ok 26 - delayed_accept

not ok 27 - dlerror

# **exit code 134**

# Output from process `dlerror`:

# Assertion failed in test/test-dlerror.c on line 45: strstr(msg, path) != NULL

ok 28 - eintr_handling

ok 29 - embed

not ok 30 - emfile

# timeout

# Output from process `emfile`: (no output)

ok 31 - env_vars

ok 32 - error_message

ok 33 - fork_fs_events_child # SKIP Filesystem watching not supported on this platform.

ok 34 - fork_fs_events_child_dir # SKIP Filesystem watching not supported on this platform.

ok 35 - fork_fs_events_file_parent_child # SKIP Filesystem watching not supported on this platform.

ok 36 - fork_signal_to_child

ok 37 - fork_signal_to_child_closed

ok 38 - fork_socketpair

ok 39 - fork_socketpair_started

ok 40 - fork_threadpool_queue_work_simple

ok 41 - fork_timer

ok 42 - fs_access

ok 43 - fs_async_dir

ok 44 - fs_async_sendfile

ok 45 - fs_chmod

ok 46 - fs_chown

ok 47 - fs_copyfile

ok 48 - fs_event_close_in_callback # SKIP Filesystem watching not supported on this platform.

ok 49 - fs_event_close_with_pending_event # SKIP Filesystem watching not supported on this platform.

ok 50 - fs_event_error_reporting

ok 51 - fs_event_getpath # SKIP Filesystem watching not supported on this platform.

ok 52 - fs_event_immediate_close # SKIP Filesystem watching not supported on this platform.

ok 53 - fs_event_no_callback_after_close # SKIP Filesystem watching not supported on this platform.

ok 54 - fs_event_no_callback_on_close # SKIP Filesystem watching not supported on this platform.

ok 55 - fs_event_ref # SKIP Filesystem watching not supported on this platform.

ok 56 - fs_event_start_and_close # SKIP Filesystem watching not supported on this platform.

ok 57 - fs_event_watch_dir # SKIP Filesystem watching not supported on this platform.

ok 58 - fs_event_watch_dir_recursive # SKIP Recursive directory watching not supported on this platform.

ok 59 - fs_event_watch_file # SKIP Filesystem watching not supported on this platform.

ok 60 - fs_event_watch_file_current_dir # SKIP Filesystem watching not supported on this platform.

ok 61 - fs_event_watch_file_exact_path # SKIP Filesystem watching not supported on this platform.

ok 62 - fs_event_watch_file_twice # SKIP Filesystem watching not supported on this platform.

ok 63 - fs_event_watch_invalid_path # SKIP Filesystem watching not supported on this platform.

ok 64 - fs_file_async

ok 65 - fs_file_loop

ok 66 - fs_file_nametoolong

ok 67 - fs_file_noent

ok 68 - fs_file_open_append

ok 69 - fs_file_pos_after_op_with_offset

ok 70 - fs_file_sync

ok 71 - fs_file_write_null_buffer

ok 72 - fs_fstat

ok 73 - fs_futime # SKIP futime not supported on Cygwin

ok 74 - fs_mkdtemp

ok 75 - fs_null_req

ok 76 - fs_open_dir

ok 77 - fs_partial_read

ok 78 - fs_partial_write

ok 79 - fs_poll

ok 80 - fs_poll_getpath

ok 81 - fs_poll_ref

ok 82 - fs_read_dir

ok 83 - fs_read_file_eof # SKIP Cygwin pread at EOF may (incorrectly) return data!

ok 84 - fs_read_write_null_arguments

ok 85 - fs_readlink

ok 86 - fs_realpath

ok 87 - fs_rename_to_existing_file

ok 88 - fs_scandir_empty_dir

ok 89 - fs_scandir_file

ok 90 - fs_scandir_non_existent_dir

ok 91 - fs_stat_missing_path

ok 92 - fs_symlink

ok 93 - fs_symlink_dir

ok 94 - fs_unlink_readonly

not ok 95 - fs_utime

# **exit code 134**
# Output from process `fs_utime`:

# Assertion failed in test/test-fs.c on line 2271: r == 0

ok 96 - fs_write_alotof_bufs

ok 97 - fs_write_alotof_bufs_with_offset

ok 98 - fs_write_multiple_bufs

ok 99 - get_currentexe

ok 100 - get_loadavg

ok 101 - get_memory

ok 102 - get_osfhandle_valid_handle

ok 103 - get_passwd

ok 104 - getaddrinfo_basic

ok 105 - getaddrinfo_basic_sync

ok 106 - getaddrinfo_concurrent

ok 107 - getaddrinfo_fail

ok 108 - getaddrinfo_fail_sync

ok 109 - gethostname

ok 110 - getnameinfo_basic_ip4

ok 111 - getnameinfo_basic_ip4_sync

ok 112 - getnameinfo_basic_ip6

not ok 113 - getsockname_tcp

# timeout

# Output from process `getsockname_tcp`:

# server socket: 0.0.0.0:9123

# accepted socket: 127.0.0.1:9123

# accepted socket peer: 127.0.0.1:61253

ok 114 - getsockname_udp

ok 115 - getters_setters

ok 116 - handle_fileno

ok 117 - handle_type_name

ok 118 - has_ref

ok 119 - homedir

ok 120 - hrtime

ok 121 - idle_ref

ok 122 - idle_starvation

ok 123 - idna_toascii

ok 124 - ip4_addr

ok 125 - ip6_addr_link_local # SKIP FIXME: This test needs more investigation on Cygwin

ok 126 - ip6_pton

ok 127 - ipc_closed_handle

ok 128 - ipc_heavy_traffic_deadlock_bug

ok 129 - ipc_listen_after_write # SKIP Cygwin runtime does not support sending handles on pipes.

ok 130 - ipc_listen_before_write # SKIP Cygwin runtime does not support sending handles on pipes.

ok 131 - ipc_send_recv_pipe # SKIP Cygwin runtime does not support sending handles on pipes.

ok 132 - ipc_send_recv_pipe_inprocess # SKIP Cygwin runtime does not support sending handles on pipes.

ok 133 - ipc_send_recv_tcp # SKIP Cygwin runtime does not support sending handles on pipes.

ok 134 - ipc_send_recv_tcp_inprocess # SKIP Cygwin runtime does not support sending handles on pipes.

ok 135 - ipc_tcp_connection # SKIP Cygwin runtime does not support sending handles on pipes.

not ok 136 - kill

# timeout

# Output from process `kill`: (no output)

ok 137 - kill_invalid_signum

ok 138 - loop_alive

ok 139 - loop_backend_timeout

ok 140 - loop_close

ok 141 - loop_configure

ok 142 - loop_handles

ok 143 - loop_instant_close

ok 144 - loop_stop

ok 145 - loop_update_time

ok 146 - multiple_listen

ok 147 - open_osfhandle_valid_handle

ok 148 - pipe_bind_error_addrinuse

ok 149 - pipe_bind_error_addrnotavail

ok 150 - pipe_bind_error_inval

ok 151 - pipe_close_stdout_read_stdin

ok 152 - pipe_connect_bad_name

ok 153 - pipe_connect_multiple # SKIP Cygwin runtime hangs on listen+connect in same process.

ok 154 - pipe_connect_on_prepare

ok 155 - pipe_connect_to_file

ok 156 - pipe_getsockname # SKIP Cygwin runtime hangs on listen+connect in same process.

ok 157 - pipe_getsockname_abstract

ok 158 - pipe_getsockname_blocking

ok 159 - pipe_listen_without_bind # SKIP Cygwin runtime hangs on listen+connect in same process.

ok 160 - pipe_pending_instances

not ok 161 - pipe_ping_pong

# timeout

# Output from process `pipe_ping_pong`: (no output)

Output from process `pipe_ping_pong`:

# PING

not ok 162 - pipe_ping_pong_vec

# timeout

# Output from process `pipe_ping_pong_vec`: (no output)

Output from process `pipe_ping_pong_vec`:

# PING

ok 163 - pipe_ref

ok 164 - pipe_ref2

ok 165 - pipe_ref3

ok 166 - pipe_ref4

ok 167 - pipe_sendmsg # SKIP Cygwin runtime does not support sending handles on pipes.

ok 168 - pipe_server_close # SKIP Cygwin runtime hangs on listen+connect in same process.

ok 169 - pipe_set_chmod

ok 170 - pipe_set_non_blocking

ok 171 - poll_bad_fdtype

ok 172 - poll_close

ok 173 - poll_close_doesnt_corrupt_stack # SKIP Test only relevant on Windows

ok 174 - poll_closesocket # SKIP Test only relevant on Windows

ok 175 - poll_duplex # SKIP Cygwin runtime hangs on listen+connect in same process.

not ok 176 - poll_oob

# timeout

# Output from process `poll_oob`: (no output)

ok 177 - poll_unidirectional # SKIP Cygwin runtime hangs on listen+connect in same process.

ok 178 - prepare_ref

ok 179 - process_priority

ok 180 - process_ref

ok 181 - process_title # SKIP uv_(get|set)_process_title is not implemented.

ok 182 - process_title_threadsafe # SKIP uv_(get|set)_process_title is not implemented.

ok 183 - queue_foreach_delete

ok 184 - ref

ok 185 - req_type_name

ok 186 - run_nowait

ok 187 - run_once

ok 188 - semaphore_1

ok 189 - semaphore_2

ok 190 - semaphore_3

ok 191 - shutdown_close_pipe

ok 192 - shutdown_close_tcp

not ok 193 - shutdown_eof

# exit code 134

# Output from process `shutdown_eof`: (no output)

Output from process `shutdown_eof`:

# Assertion failed in test/test-shutdown-eof.c on line 112: got_q

ok 194 - shutdown_twice

ok 195 - signal_multiple_loops # SKIP FIXME: This test needs more investigation on Cygwin

ok 196 - socket_buffer_size

ok 197 - spawn_and_kill

ok 198 - spawn_and_kill_with_std

ok 199 - spawn_and_ping

ok 200 - spawn_auto_unref

ok 201 - spawn_closed_process_io

ok 202 - spawn_detached

ok 203 - spawn_exit_code

ok 204 - spawn_fails

ok 205 - spawn_fails_check_for_waitpid_cleanup

ok 206 - spawn_fs_open

ok 207 - spawn_ignored_stdio

ok 208 - spawn_inherit_streams

ok 209 - spawn_preserve_env

ok 210 - spawn_quoted_path # SKIP Test for Windows

ok 211 - spawn_reads_child_path # SKIP       0 [main] run-tests 1276 C:\cygwin64\home\damon\projects
\c\libuv\test\.libs\run-tests.exe: *** fatal error - Internal error: TP_NUM_W_BUFS too small: 50.

ok 212 - spawn_setgid_fails

ok 213 - spawn_setuid_fails

ok 214 - spawn_setuid_setgid # SKIP It should be run as root user

ok 215 - spawn_stdin

ok 216 - spawn_stdio_greater_than_3

ok 217 - spawn_stdout

ok 218 - spawn_stdout_and_stderr_to_file

ok 219 - spawn_stdout_and_stderr_to_file2

ok 220 - spawn_stdout_and_stderr_to_file_swap

ok 221 - spawn_stdout_to_file

ok 222 - spawn_tcp_server

ok 223 - stdio_over_pipes

ok 224 - sys_error

not ok 225 - tcp6_ping_pong

# timeout

# Output from process `tcp6_ping_pong`: (no output)

Output from process `tcp6_ping_pong`:

# PING

not ok 226 - tcp6_ping_pong_vec

# timeout

# Output from process `tcp6_ping_pong_vec`: (no output)

Output from process `tcp6_ping_pong_vec`:

# PING

not ok 227 - tcp_alloc_cb_fail

# timeout

# Output from process `tcp_alloc_cb_fail`: (no output)

ok 228 - tcp_bind6_error_addrinuse

ok 229 - tcp_bind6_error_addrnotavail

ok 230 - tcp_bind6_error_fault

ok 231 - tcp_bind6_error_inval

ok 232 - tcp_bind6_localhost_ok

ok 233 - tcp_bind_error_addrinuse

ok 234 - tcp_bind_error_addrnotavail_1

ok 235 - tcp_bind_error_addrnotavail_2

ok 236 - tcp_bind_error_fault

ok 237 - tcp_bind_error_inval

ok 238 - tcp_bind_invalid_flags

ok 239 - tcp_bind_localhost_ok

ok 240 - tcp_bind_writable_flags

ok 241 - tcp_close

not ok 242 - tcp_close_accept

# timeout

# Output from process `tcp_close_accept`: (no output)

not ok 243 - tcp_close_while_connecting

# timeout

# Output from process `tcp_close_while_connecting`: (no output)

ok 244 - tcp_connect_error_after_write

ok 245 - tcp_connect_error_fault

not ok 246 - tcp_connect_timeout

# timeout

# Output from process `tcp_connect_timeout`: (no output)

ok 247 - tcp_create_early

ok 248 - tcp_create_early_accept

ok 249 - tcp_create_early_bad_bind

ok 250 - tcp_create_early_bad_domain

ok 251 - tcp_flags

ok 252 - tcp_listen_without_bind

not ok 253 - tcp_oob

# timeout

# Output from process `tcp_oob`: (no output)

not ok 254 - tcp_open

# timeout

# Output from process `tcp_open`: (no output)

Output from process `tcp_open`: (no output)

ok 255 - tcp_open_bound

not ok 256 - tcp_open_connected

# timeout

# Output from process `tcp_open_connected`: (no output)

Output from process `tcp_open_connected`: (no output)

ok 257 - tcp_open_twice

not ok 258 - tcp_ping_pong

# timeout

# Output from process `tcp_ping_pong`: (no output)

Output from process `tcp_ping_pong`:

# PING

not ok 259 - tcp_ping_pong_vec

# timeout

# Output from process `tcp_ping_pong_vec`: (no output)

Output from process `tcp_ping_pong_vec`:

# PING

ok 260 - tcp_read_stop

ok 261 - tcp_ref

ok 262 - tcp_ref2

ok 263 - tcp_ref2b

ok 264 - tcp_ref3

ok 265 - tcp_ref4

ok 266 - tcp_shutdown_after_write

not ok 267 - tcp_try_write

# timeout

# Output from process `tcp_try_write`: (no output)

not ok 268 - tcp_unexpected_read

# timeout

# Output from process `tcp_unexpected_read`: (no output)

ok 269 - tcp_write_after_connect

ok 270 - tcp_write_fail

not ok 271 - tcp_write_queue_order

# timeout

# Output from process `tcp_write_queue_order`: (no output)

not ok 272 - tcp_write_ready

# timeout

# Output from process `tcp_write_ready`: (no output)

Output from process `tcp_write_ready`: (no output)

not ok 273 - tcp_write_to_half_open_connection

# timeout

# Output from process `tcp_write_to_half_open_connection`: (no output)

not ok 274 - tcp_writealot

# timeout

# Output from process `tcp_writealot`: (no output)

Output from process `tcp_writealot`: (no output)

ok 275 - thread_create

ok 276 - thread_equal

ok 277 - thread_local_storage

ok 278 - thread_mutex

ok 279 - thread_mutex_recursive

ok 280 - thread_rwlock

ok 281 - thread_rwlock_trylock

ok 282 - thread_stack_size

ok 283 - threadpool_cancel_fs

ok 284 - threadpool_cancel_getaddrinfo

ok 285 - threadpool_cancel_getnameinfo

ok 286 - threadpool_cancel_single

ok 287 - threadpool_cancel_work

ok 288 - threadpool_multiple_event_loops

ok 289 - threadpool_queue_work_einval

ok 290 - threadpool_queue_work_simple

ok 291 - timer

ok 292 - timer_again

ok 293 - timer_early_check

ok 294 - timer_from_check

ok 295 - timer_huge_repeat

ok 296 - timer_huge_timeout

ok 297 - timer_init

ok 298 - timer_null_callback

ok 299 - timer_order

ok 300 - timer_ref

ok 301 - timer_ref2

ok 302 - timer_run_once

ok 303 - timer_start_twice

ok 304 - tmpdir

ok 305 - tty

ok 306 - tty_file

ok 307 - tty_pty

ok 308 - udp_alloc_cb_fail

ok 309 - udp_bind

ok 310 - udp_bind_reuseaddr

ok 311 - udp_create_early

ok 312 - udp_create_early_bad_bind

ok 313 - udp_create_early_bad_domain

ok 314 - udp_dgram_too_big

ok 315 - udp_dual_stack # SKIP FIXME: This test needs more investigation on Cygwin

ok 316 - udp_ipv6_only

ok 317 - udp_multicast_interface

ok 318 - udp_multicast_interface6

ok 319 - udp_multicast_join

ok 320 - udp_multicast_join6

ok 321 - udp_multicast_ttl

ok 322 - udp_no_autobind

ok 323 - udp_open

ok 324 - udp_open_twice

ok 325 - udp_options

ok 326 - udp_options6

ok 327 - udp_ref

ok 328 - udp_ref2

ok 329 - udp_ref3

ok 330 - udp_send_and_recv

ok 331 - udp_send_hang_loop

ok 332 - udp_send_immediate

ok 333 - udp_send_unreachable

ok 334 - udp_try_send

ok 335 - unref_in_prepare_cb

ok 336 - utf8_decode1

ok 337 - walk_handles

ok 338 - watcher_cross_stop

ok 339 - we_get_signal

ok 340 - we_get_signal_one_shot

ok 341 - we_get_signals

ok 342 - we_get_signals_mixed

FAIL: test/run-tests.exe

======================================================

1 of 1 test failed

Please report to https://github.com/libuv/libuv/issues

======================================================

make[1]: *** [Makefile:4668: check-TESTS] Error 1

make[1]: Leaving directory '/home/damon/projects/c/libuv'

make: *** [Makefile:4925: check-am] Error 2

make[1]: Entering directory '/home/damon/projects/c/libuv'

 /usr/bin/mkdir -p '/home/damon/.local/lib'

 /bin/sh ./libtool   --mode=install /usr/bin/install -c   libuv.la '/home/damon/.local/lib'

libtool: install: /usr/bin/install -c .libs/libuv.dll.a /home/damon/.local/lib/libuv.dll.a

libtool: install: base_file=`basename libuv.la`

libtool: install:  dlpath=`/bin/sh 2>&1 -c '. .libs/'libuv.la'i; echo cyguv-1.dll'`

libtool: install:  dldir=/home/damon/.local/lib/`dirname ../bin/cyguv-1.dll`

libtool: install:  test -d /home/damon/.local/lib/../bin || mkdir -p /home/damon/.local/lib/../bin

libtool: install:  /usr/bin/install -c .libs/cyguv-1.dll /home/damon/.local/lib/../bin/cyguv-1.dll

libtool: install:  chmod a+x /home/damon/.local/lib/../bin/cyguv-1.dll

libtool: install:  if test -n '' && test -n 'strip --strip-unneeded'; then eval 'strip --strip-unneeded 

/home/damon/.local/lib/../bin/cyguv-1.dll' || exit 0; fi

libtool: install: /usr/bin/install -c .libs/libuv.lai /home/damon/.local/lib/libuv.la

libtool: install: /usr/bin/install -c .libs/libuv.a /home/damon/.local/lib/libuv.a

libtool: install: chmod 644 /home/damon/.local/lib/libuv.a

libtool: install: ranlib /home/damon/.local/lib/libuv.a

 /usr/bin/mkdir -p '/home/damon/.local/include'

 /usr/bin/install -c -m 644 include/uv.h '/home/damon/.local/include'

 /usr/bin/mkdir -p '/home/damon/.local/lib/pkgconfig'

 /usr/bin/install -c -m 644 libuv.pc '/home/damon/.local/lib/pkgconfig'

 /usr/bin/mkdir -p '/home/damon/.local/include/uv'

 /usr/bin/install -c -m 644 include/uv/errno.h include/uv/threadpool.h include/uv/version.h include/uv

/unix.h include/uv/posix.h '/home/damon/.local/include/uv'

make[1]: Leaving directory '/home/damon/projects/c/libuv'
```
</details>

**Comment by bnoordhuis** [comment]
Thanks for the report. Cygwin is not a supported platform/environment and not tested so YMMV when using libuv on cygwin. We'll accept pull requests that fix test failures if you have time to look into them.

**Comment by damon-kwok** [comment]
cygwin test fix:uv_resident_set_memory #2081
